addons/hr_expense/models/hr_expense_sheet.py

```diff
@@ -693,19 +701,17 @@ def _do_create_moves(self):
 
         for sheet in own_account_sheets:
             sheet.accounting_date = sheet.accounting_date or sheet._calculate_default_accounting_date()
-        moves = self.env['account.move'].create([sheet._prepare_bills_vals() for sheet in own_account_sheets])
+        moves = self.env['account.move'].sudo().create([sheet._prepare_bills_vals() for sheet in own_account_sheets])
```

Comment:
```suggestion
        moves_in_sudo = self.env['account.move'].sudo().create([sheet._prepare_bills_vals() for sheet in own_account_sheets])
```
I like to make it explicit. Feel free to ignore, tho, or to keep it for another PR